### ISSUE TITLE
Incompatible versions of undercover and  simplecov-lcov gems

### ISSUE THREAD

**Issue by korrs** [grodowski/pronto-undercover#16]
When I run command ‘pronto run’ i’m getting an error: 
```
undercover-0.3.4/lib/undercover/lcov_parser.rb:47:in `parse_line': could not recognise 'BRF:0 (Undercover::LcovParseError)
```
I found that [version of undercover gem](https://github.com/grodowski/pronto-undercover/blob/master/pronto-undercover.gemspec#L24) not matched with the last version of simplecov-lcov gem witch generate report [with](https://github.com/fortissimo1997/simplecov-lcov/blob/master/lib/simplecov-lcov.rb#L93) the ‘BRF’ symbols. But underсover of 0.3.4 version doesn’t expect this format of coverage report.  

Obviously it can be fixed it by specifying the correct version of underсover in the .gemspec file.

**Comment by grodowski** [comment]
Thank you Vladislav! I'm going to merge your PR soon. The minor issue with newer `undercover` vs. `pronto-undercover` is that there's no way (yet) to incorporate branch coverage into warning messages (unless someone added it and I forgot!)

**Comment by benjamin-ltr** [comment]
I'm having the same issue. Is it possible to release a new version to fix the issue?
Thanks a lot

**Comment by grodowski** [comment]
Just released `0.2.0` 🎉

**Comment by benjamin-ltr** [comment]
Awesome, thanks for being so reactive
